src/main/java/net/greatstart/controllers/ProjectRestController.java

```diff
@@ -0,0 +1,81 @@
+package net.greatstart.controllers;
+
+import net.greatstart.dto.DtoProject;
+import net.greatstart.mappers.ProjectMapper;
+import net.greatstart.model.Project;
+import net.greatstart.services.CategoryService;
+import net.greatstart.services.ProjectService;
+import net.greatstart.services.UserService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.util.Collection;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/project")
+public class ProjectRestController {
+    private static final String REDIRECT_TO_PROJECTS = "redirect:/project/";
+    private static final String PROJECTS = "project/projects";
+    private static final String PROJECT = "dtoProject";
+
+    private final Logger logger = LoggerFactory.getLogger(ProjectRestController.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "logger" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)